FinnUI/Sources/Fullscreen/MotorTransactionInsurancePickerView/MotorTransactionInsuranceView.swift

```diff
@@ -92,3 +99,11 @@ extension MotorTransactionInsuranceView: NavigationLinkViewDelegate {
         delegate?.motorTransactionInsuranceViewWasSelected(self)
     }
 }
+
+// MARK: - Private extensions
+
+private extension CGColor {
+    static var logoImageBorderColor: CGColor {
+        UIColor.dynamicColorIfAvailable(defaultColor: .sardine, darkModeColor: .darkSardine).cgColor
```

Comment:
We already have the semantic color `imageBorder` that's equal to this. Can we use that?

Reply:
Nice, wasn't aware of that one 👍